tonic-build/src/server.rs

```diff
@@ -375,8 +375,14 @@ fn generate_named(
     service_name: &str,
 ) -> TokenStream {
     let service_name = syn::LitStr::new(service_name, proc_macro2::Span::call_site());
+    let name_doc = generate_doc_comment(" Generated gRPC service name");
 
     quote! {
+        #name_doc
+        pub const fn service_name() -> &'static str {
```

Comment:
I think it would still make sense for this to a `const SERVICE_NAME: &str`?

Reply:
fair point, should the trait const below be `const NAME: &'static str = SERVICE_NAME;` then?

Reply:
Might be nice, want to make that change?